pkgs/applications/graphics/processing/default.nix

```diff
@@ -1,73 +1,85 @@
-{ lib, stdenv, fetchFromGitHub, fetchpatch, fetchurl, xmlstarlet, makeWrapper, ant, jdk, rsync, javaPackages, libXxf86vm, gsettings-desktop-schemas }:
+{ lib, stdenv, fetchFromGitHub, fetchurl, ant, unzip, makeWrapper, jdk11, javaPackages, rsync, ffmpeg, batik, gsettings-desktop-schemas, xorg, wrapGAppsHook }:
+let
+  vaqua = fetchurl {
+    name = "VAqua9.jar";
+    url = "https://violetlib.org/release/vaqua/9/VAqua9.jar";
+    sha256 = "cd0b82df8e7434c902ec873364bf3e9a3e6bef8b59cbf42433130d71bf1a779c";
+  };
+
+  jna = fetchurl {
+    name = "jna-5.8.0.zip";
+    url = "https://github.com/java-native-access/jna/archive/5.8.0.zip";
+    sha256 = "7cRkkxyo2Ld19Nif8oWceITt7dYMUnQlrs0R5e0AmPE=";
+  };
+
+  gluon = fetchurl {
```

Comment:
Is there a reason this cannot use the nixpkgs `openjfx` packages [here](https://github.com/NixOS/nixpkgs/tree/nixos-21.05/pkgs/development/compilers/openjdk/openjfx)?